net/dynapoint/Makefile

```diff
@@ -0,0 +1,49 @@
+#
+# Copyright (C) 2016 LEDE project (Tobias Ilte)
+#
+# This is free software, licensed under the GNU General Public License v3.
```

Comment:
I guess you mixed the license for the Makefile itself and the license
for the payload. The license for the Makefile should be GPLv2, so
I prefer to see here the common header:

# This is free software, licensed under the GNU General Public License v2.
# See /LICENSE for more information.